### PR TITLE
add computed disabled and add action column for each Document editor

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/doc-editors/ConfigTemplatesEditor.vue [hunk 1]
```diff
@@ -584,6 +589,20 @@ export default defineComponent({
       return `${apiPrefix}/reblaze/${apiPath}`
     },
 
+    selectedDocNotDeletable(): boolean {
+      return !this.selectedConfigTemplate ||
+        this.selectedConfigTemplate.id.startsWith('__') || // Default entries
+        this.selectedConfigTemplate.id.startsWith('rl-') || // Reblaze-managed Rate Limits
```
Comment (Aviv-Galmidi) [inline src/doc-editors/ConfigTemplatesEditor.vue:595]: We don't need `rl-`, `action-`, `rbz-`, or `dr_` in this file

File: src/doc-editors/MobileSDKsEditor.vue [hunk 2]
```diff
@@ -402,6 +406,20 @@ export default defineComponent({
       return `${apiPrefix}/reblaze/configs/${this.selectedBranch}/d/mobile-sdks/e/${this.selectedMobileSDK.id}/`
     },
 
+    selectedDocNotDeletable(): boolean {
+      return !this.selectedMobileSDK ||
+        this.selectedMobileSDK.id.startsWith('__') || // Default entries
+        this.selectedMobileSDK.id.startsWith('rl-') || // Reblaze-managed Rate Limits
```
Comment (Aviv-Galmidi) [inline src/doc-editors/MobileSDKsEditor.vue:412]: We don't need `rl-`, `action-`, `rbz-`, or `dr_` in this file

File: src/doc-editors/MobileSDKsEditor.vue [hunk 3]
```diff
@@ -590,6 +608,19 @@ export default defineComponent({
...
+    async loadReferencedMobileSDKsIDs() {
+      const response = await RequestsUtils.sendReblazeRequest({
+        methodName: 'GET',
+        url: `configs/${this.selectedBranch}/d/sites/`,
+      })
+      const serverGroups = response?.data || []
+      const referencedMobileSDKs: string[] = []
+      _.forEach(serverGroups, (serverGroup) => {
+        referencedMobileSDKs.push(serverGroup['mobile_sdk'])
+      })
```
Comment (Aviv-Galmidi) [inline src/doc-editors/MobileSDKsEditor.vue:621]: `const referencedMobileSDKs = _.map(serverGroups, 'mobile_sdk')`

File: src/doc-editors/ConfigTemplatesEditor.vue [hunk 4]
```diff
@@ -674,6 +693,19 @@ export default defineComponent({
...
+    async loadReferencedConfigTemplatesIDs() {
+      const response = await RequestsUtils.sendReblazeRequest({
+        methodName: 'GET',
+        url: `configs/${this.selectedBranch}/d/sites/`,
+      })
+      const serverGroups = response?.data || []
+      const referencedConfigTemplates: string[] = []
+      _.forEach(serverGroups, (serverGroup) => {
+        referencedConfigTemplates.push(serverGroup['proxy_template'])
+      })
```
Comment (Aviv-Galmidi) [inline src/doc-editors/ConfigTemplatesEditor.vue:705]: `referencedConfigTemplates = _.map(serverGroups, 'proxy_template')`

File: src/doc-editors/RoutingProfilesEditor.vue [hunk 5]
```diff
@@ -406,6 +410,20 @@ export default defineComponent({
       return `${apiPrefix}/reblaze/configs/${this.selectedBranch}/d/routing-profiles/e/${selectedId}/`
     },
 
+    selectedDocNotDeletable(): boolean {
+      return !this.selectedRoutingProfile ||
+        this.selectedRoutingProfile.id.startsWith('__') || // Default entries
+        this.selectedRoutingProfile.id.startsWith('rl-') || // Reblaze-managed Rate Limits
```
Comment (Aviv-Galmidi) [inline src/doc-editors/RoutingProfilesEditor.vue:416]: We don't need `rl-`, `action-`, `rbz-`, or `dr_` in this file

File: src/doc-editors/RoutingProfilesEditor.vue [hunk 6]
```diff
@@ -611,6 +629,19 @@ export default defineComponent({
...
+    async loadReferencedRoutingProfilesIDs() {
+      const response = await RequestsUtils.sendReblazeRequest({
+        methodName: 'GET',
+        url: `configs/${this.selectedBranch}/d/sites/`,
+      })
+      const serverGroups = response?.data || []
+      const referencedRoutingProfiles: string[] = []
+      _.forEach(serverGroups, (serverGroup) => {
+        referencedRoutingProfiles.push(serverGroup['routing_profile'])
+      })
```
Comment (Aviv-Galmidi) [inline src/doc-editors/RoutingProfilesEditor.vue:642]: `const referencedRoutingProfiles = _.map(serverGroups, 'routing_profile')`

File: src/views/DocumentEditor.vue [hunk 7]
```diff
@@ -684,7 +696,81 @@ export default defineComponent({
...
+    async loadReferencedSecurityPoliciesIDs() {
+      const response = await RequestsUtils.sendReblazeRequest({
+        methodName: 'GET',
+        url: `configs/${this.selectedBranch}/d/sites/`,
+      })
+      const serverGroups = response?.data || []
+      const referencedSecurityPolicies: string[] = []
+      _.forEach(serverGroups, (serverGroup) => {
+        referencedSecurityPolicies.push(serverGroup['security_policy'])
+      })
```
Comment (Aviv-Galmidi) [inline src/views/DocumentEditor.vue:772]: `const referencedSecurityPolicies = _.map(serverGroups, 'security_policy')`

File: src/doc-editors/ConfigTemplatesEditor.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/doc-editors/ConfigTemplatesEditor.vue:595]: Done

File: src/doc-editors/MobileSDKsEditor.vue (hunk 2 above)
Comment (AvihayMReblaze) [reply src/doc-editors/MobileSDKsEditor.vue:412]: Done

File: src/doc-editors/MobileSDKsEditor.vue (hunk 3 above)
Comment (AvihayMReblaze) [reply src/doc-editors/MobileSDKsEditor.vue:621]: Done

File: src/doc-editors/ConfigTemplatesEditor.vue (hunk 4 above)
Comment (AvihayMReblaze) [reply src/doc-editors/ConfigTemplatesEditor.vue:705]: Done

File: src/doc-editors/RoutingProfilesEditor.vue (hunk 5 above)
Comment (AvihayMReblaze) [reply src/doc-editors/RoutingProfilesEditor.vue:416]: Done

File: src/doc-editors/RoutingProfilesEditor.vue (hunk 6 above)
Comment (AvihayMReblaze) [reply src/doc-editors/RoutingProfilesEditor.vue:642]: Done

File: src/views/DocumentEditor.vue (hunk 7 above)
Comment (AvihayMReblaze) [reply src/views/DocumentEditor.vue:772]: Done

**Pull request merged**